### PR TITLE
Add space on info message using %d

### PR DESCRIPTION
**Pull request by panizzag**
At the end of successful execution, the logger component does not show the number of records processed;

`May 9th 2020, 15:51:42.507	SUCCESSFULLY EXECUTED BATCH OPERATION. TOTAL EXECUTED STATEMENTS: %d.`

### CODE REVIEW

File: src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ private void logBulkUpdateInfo(int[] updateCounts, int batchCount) {
           .error("BULK UPDATE EXCEPTION. DATABASE PROCESSED %d OPERATIONS SUCCESSFULLY AND STOPPED PROCESSING DUE TO EXCEPTION.",
                  successfulOperations + noInfoAvailable);
     } else {
-      LOGGER.info("SUCCESSFULLY EXECUTED BATCH OPERATION. TOTAL EXECUTED STATEMENTS: %d.", batchCount);
+      LOGGER.info("SUCCESSFULLY EXECUTED BATCH OPERATION. TOTAL EXECUTED STATEMENTS: %d .", batchCount);
```
Comment (emariotti3) [inline src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java:125]: Change %d for {}

Comment (panizzag) [reply src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java]: Done

File: src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java [hunk 2]
```diff
@@ -122,7 +122,7 @@ private void logBulkUpdateInfo(int[] updateCounts, int batchCount) {
           .error("BULK UPDATE EXCEPTION. DATABASE PROCESSED %d OPERATIONS SUCCESSFULLY AND STOPPED PROCESSING DUE TO EXCEPTION.",
                  successfulOperations + noInfoAvailable);
     } else {
-      LOGGER.info("SUCCESSFULLY EXECUTED BATCH OPERATION. TOTAL EXECUTED STATEMENTS: %d.", batchCount);
+      LOGGER.info("SUCCESSFULLY EXECUTED BATCH OPERATION. TOTAL EXECUTED STATEMENTS: {}.", batchCount);
```
[the remark this answers is not in the document]
Comment (panizzag) [reply src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java]: Hi Steve, I've replaced the logger.

**Comment by julodiaz** [comment]
Closing because #287 is more complete. Changing **info** for **debug** as Steve requested.

**Pull request closed without merging**